regular_expr.py

```diff
@@ -0,0 +1,34 @@
+"""Regular Expressions."""
+import re
+
+# Write a program to find all email addresses in string.
+s1 = 'My email is natallia_haisionak@epam.com '
+result = re.findall(r'\w+@\w+\.\w+', s1)
+print(result)
+
+# Write a program to find all 3, 4, 5 characters long words in a string.
+s2 = 'London is the capital and most populous city of England and the UK!'
+result = re.findall(r'\w+', s2)
+for i in result:
+    if len(i) in (3, 4, 5):
```

Comment:
you can use {3,5} instruction instead in your regular expression. It's just a tip.